src/main/resources/changelog/scripts/v-1.0.0/resource_graph/migration/resources_date_fields.sql

```diff
@@ -0,0 +1,12 @@
+alter table resources
+  add column created_date timestamp default current_timestamp not null,
```

Comment:
As created_date is a "not null" field, should we add a update script to set current_timestamp for all existing records? Else, upgrade will fail for sprint testing env right?

Another option is to ask Oleksandr Haimanov if he can delete mod-linked-data from sprint testing environment so that we can start from scratch. Let me send him a message including you.

Reply:
See Oleksandr's response in slack. This is not a problem. We have to open a ticket to delete mod-linked-data from the environments.

Also, we do not have  to create alter_table SQL. Instead, we can include it in the Create table script as you originally did.

Reply:
have reverted the changes